element.io/nightly/config.json

```diff
@@ -49,5 +49,6 @@
         "feature_spotlight": true,
         "feature_video_rooms": true
     },
-    "map_style_url": "https://api.maptiler.com/maps/streets/style.json?key=fU3vlMsMn4Jb6dnEIFsx"
+    "map_style_url": "https://api.maptiler.com/maps/streets/style.json?key=fU3vlMsMn4Jb6dnEIFsx",
+    "tray_icon_path": null
```

Comment:
we shouldn't be defining `null` when trying to use default values

Reply:
How come, actually?

Either way, the default is now `{}` as per the other change.